test/resources/oc.resource

```diff
@@ -61,10 +61,18 @@ Oc Exec
     RETURN    ${output}
 
 Oc Wait
-    [Documentation]    Run 'oc wait' on a specific resource in the current test namespace
+    [Documentation]    Run 'oc wait' on a specific resource in the current test namespace.
```

Comment:
I would note here that resource must include the namespace. Or make the namespace a separate argument.

Reply:
Reworded the comment.